### PR TITLE
(ungoogled-chromium) Migrate package

### PR DESCRIPTION
**Pull request by TakataSanshiro**
## Description
embedded and automated ungoogled-chromium.

## Motivation and Context
automation is needed since ungoogled-chromium is updated frequently.

## How Has this Been Tested?
Chocolatey testing environment.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by pauby** [comment]
I don't see an issue or discussion for this. Did you follow the [migration guidelines](https://github.com/chocolatey-community/chocolatey-packages/wiki/Package-migration-process)?

**Comment by TakataSanshiro** [comment]
I didn't see that. I'll open a new discussion.

**Comment by AdmiringWorm** [comment]
Moving this PR into draft until we a discussion has been created, and we have decided whether to accept the package into this repository.

**Comment by AdmiringWorm** [comment]
Moving to draft until user has done as asked in the created discussion #2119

**Comment by TakataSanshiro** [comment]
https://github.com/chocolatey-community/chocolatey-packages/issues/2213

File: automatic/ungoogled-chromium/update.ps1 [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "(?i)(^\s*file64\s*=\s*`"[$]toolsDir\\).*" = "`${1}$($Latest.FileName64)`""
+        }
+        "ungoogled-chromium.nuspec" = @{
+          "(\<licenseUrl\>).*(\<\/licenseUrl\>)"     = "`${1}$($Latest.CopyingUrl)`$2"
+          "(\<releaseNotes\>).*(\<\/releaseNotes\>)" = "`${1}$($Latest.ChangeLog)`$2"
+        }
+    }
+}
+
+function global:au_GetLatest {
```
Comment (pauby) [inline automatic/ungoogled-chromium/update.ps1:21]: There is a `Get-GitHubRelease` function that will do a lot of this work now.

**Review by pauby — CHANGES_REQUESTED** [review]
This is using the Zip install which is just putting this files into the tools folder. There is also an EXE installer.

Should this package not be changed to a virtual package, and a `ungoogled-chriomum.portable` package created using this code? That would allow somebody to create an `ungoogled-chromium.install` package and everything would be much cleaner.

**Comment by TakataSanshiro** [comment]
This issue has already been fixed.
> There are many times failed when pushing embedded packages from my pc.

I have no motivation to PR it anymore.

> somebody to create an ungoogled-chromium.install package
When the time has come, I change one.

**Pull request closed without merging**